openfecwebapp/api_caller.py

```diff
@@ -0,0 +1,40 @@
+import requests
+
+from openfecwebapp.local_config import api_location
+
+# api urls are singular, public urls are plural
+type_map = {
+    'candidates': 'candidate',
+    'committees': 'committee',
+    'candidate': 'candidate',
+    'committee': 'committee'
+}
```

Comment:
It looks like you're using a plural form in the __init__.py page, but immediately converting to singular here. Why not just use singular throughout? I'd also recommend using constants rather than strings.